src/Gelf/Logger.php

```diff
@@ -181,7 +181,7 @@ private static function interpolate($message, array $context)
         // build a replacement array with braces around the context keys
         $replace = array();
         foreach ($context as $key => $val) {
-            $replace['{' . $key . '}'] = $val;
+            $replace['{' . $key . '}'] = is_string($val) ? $val : 'Array';
```

Comment:
@Seldaek Would it be okay for you if we copy your implementation to gelf-php? (With the appropiate author notice of course)